### PR TITLE
New App-Bar

### PR DESCRIPTION
**Pull request by felixheidecke**
## Description
Bring over new App-Bar layout to Phoenix

## Related PRs

Only works in conjunction with https://github.com/owncloud/owncloud-design-system/pull/323

## How Has This Been Tested?

Manual test

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFavorites</strong> failed. Please find the screenshots inside ...</summary><p>












</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFolders</strong> failed. Please find the screenshots inside ...</summary><p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFiles</strong> failed. Please find the screenshots inside ...</summary><p>








</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary><p>
















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFolders</strong> failed. Please find the screenshots inside ...</summary><p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary><p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFavorites</strong> failed. Please find the screenshots inside ...</summary><p>












</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRenameFiles</strong> failed. Please find the screenshots inside ...</summary><p>








</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary><p>



















</p></details>

File: apps/files/src/components/FilesAppBar.vue [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
-        <span v-if="!this.canUpload" v-translate>You have no permission to upload.</span>
-      </div>
-      <div class="uk-navbar-item">
-        <translate :translate-n="activeFiles.length" translate-plural="%{ activeFiles.length } Results">
-          %{ activeFiles.length } Result
-        </translate>
-      </div>
-      <div class="uk-navbar-item">
-        <oc-icon name="filter_list" id="oc-filter-list-btn" />
+        <oc-drop toggle="#new-file-menu-btn" mode="hover" :options="{pos:'bottom-right'}">
```
Comment (DeepDiver1975) [inline apps/files/src/components/FilesAppBar.vue]: drop - not needed - ``` <file-filter-menu />``` holds this

File: apps/files/src/components/FilesAppBar.vue [hunk 2]
```diff
@@ -1,55 +1,52 @@
...
+        <oc-drop toggle="#oc-progress-pie" mode="click">
+          <oc-upload-menu :items="inProgress" />
+        </oc-drop>
+      </div>
+    </oc-grid>
+    <oc-dialog-prompt name="new-folder-dialog" :oc-active="createFolder" v-model="newFolderName" ocInputId="new-folder-input" ocConfirmId="new-folder-ok"
+                    :ocLoading="fileFolderCreationLoading"
                     :ocTitle="_createFolderDialogTitle" @oc-confirm="addNewFolder" @oc-cancel="createFolder = false; newFolderName = ''"></oc-dialog-prompt>
-  <oc-dialog-prompt name="new-file-dialog" :oc-active="createFile" v-model="newFileName"
-                    :ocLoading="fileFolderCreationLoading" :ocError="newFileErrorMessage"
```
Comment (DeepDiver1975) [inline apps/files/src/components/FilesAppBar.vue]: killing features again ...

File: src/Phoenix.vue [hunk 3]
```diff
@@ -1,27 +1,21 @@
 <template>
-  <div
-    id="Phoenix"
-    :class="['uk-flex uk-flex-column uk-height-1-1', { loginGradient: showGradientBackground}]"
-    :style="showImageBackground ? { 'background-image': 'url('+configuration.theme.logo.background+')',
```
Comment (DeepDiver1975) [inline src/Phoenix.vue]: becasue we no longer need these configuration values please drop them from the theme jsons files as well as from the store

File: src/Phoenix.vue [hunk 4]
```diff
@@ -41,31 +35,7 @@ export default {
     ...mapGetters(['configuration']),
     showHeader () {
       return this.$route.meta.hideHeadbar !== true
-    },
-    showBackground () {
-      if (!this.$route.meta.showBackground) {
```
Comment (DeepDiver1975) [inline src/Phoenix.vue]: showBackground is on some route definitions as well - please remove them - no longer needed

**Comment by DeepDiver1975** [comment]
no many conflict - and once again too many changes in one PR -> closing

**Pull request closed without merging**